src/subgraph/index.ts

```diff
@@ -134,4 +143,59 @@ export class Subgraph {
     const subgraphEndpoint = this.getSubgraphEndpoint(this.rpcConfig.chainId);
     return getAllVaults(subgraphEndpoint);
   }
+
+  public async getChainVaultData(): Promise<Vault[]> {
+    const subgraphEndpoint = this.getSubgraphEndpoint(this.rpcConfig.chainId);
+    return getChainVaultData(this.rpcConfig.chainId, subgraphEndpoint);
+  }
+
+  public async getTotalPoolValue() {
```

Comment:
Please add comments to all functions and lets rename it to `getTotalPoolsValue`, as its for all pools, not just one

Reply:
done